### DISCUSSION TITLE
I recently got my student benefit with my campus gmail github account but my main github account is with another gmail . Problem is how can i use my benefits in another gmail(i.e main github account)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#109954] [GitHub Education]
### Hi everyone,

Or if i will delete my account that has benefit ....will i be able to apply again for student developer pack by keeping as secondary gmail in my main github account??

**Comment by Bashmdz** [comment]
If you've received student benefits through your campus Gmail account but you want to use those benefits with your main GitHub account associated with a different Gmail address, you'll need to contact GitHub Support for assistance. They might be able to help you transfer the benefits to your main account or provide you with instructions on how to link your student benefits to your main account.

Here's what you can do:

Contact GitHub Support: Send an email to GitHub Support explaining your situation. Make sure to provide them with both your campus Gmail account and your main GitHub account details. They should be able to assist you in transferring the benefits or linking your main account to the student benefits.

Provide Proof: Be ready to provide any necessary proof that you're eligible for student benefits. This might include documentation from your educational institution confirming your student status.

Follow Their Instructions: Once you've contacted GitHub Support, they will likely guide you through the process of either transferring the benefits or linking your main account to the student benefits.

Remember to be patient as the process may take some time, but GitHub Support should be able to help you resolve this issue.

**Comment by ghost** [comment]
i have also contacted github support by creating ticket regarding this
issue but there is no response from there. How much time they take reply.??
